### ISSUE TITLE
photo adjustment processing integration

### ISSUE THREAD

**Issue by abfleishman** [Microsoft/VoTT#242]
Sometimes we are processing photos that were over-/under- exposed.  It would be nice to be able to adjust contrast, brightness, colors, (basic photo editing adjustments) while viewing files.  This would be nice to have a way to lock the change when switching images

**Comment by kichinsky** [comment]
We have an idea of visual canvas-based filters that might be applied to images, the basic infrastructure for that is implemented, but nothing exposed to the interface layer yet.

**Comment by kichinsky** [comment]
@abfleishman I added basic invertion filter as an example in the dev branch. Could you test it to check whether is it something you are looking for? I will add other visual filters then.

**Comment by abfleishman** [comment]
@kichinsky I am not yet up and running on the dev branch (something I hope to do this week)

**Comment by abfleishman** [comment]
@kichinsky I have the dev version running now and I think that the filter idea is what I am getting at, but to make it more useful I think it would need to have some keyboard integration. 
 For instance, the "-" and "+" keys to adjust brightness, the "[" and "]" keys to adjust contrast, or "shift" and "-/+" to adjust saturation. Having a quick button (keyboard shortcut) to "reset" the image would be important too!  
I think being able to quickly make adjustments is crucial.  
For instances, I am looking through 100 images and I come to a very dark one.  I want to just hit the "+" button a few times to lighten it up to check to make sure there are no objects of interest in there before moving to the next image.  I could also see having 10 photos in a row that are all dark and would want a way to persist the filter from one image to the next until I have passed through the dark images and then I could hit reset and go back to normal.

**Comment by abfleishman** [comment]
@kichinsky I just saw that the "reset filters" dropdown menu option does not work. But the filter resets if you move to the next image and come back.

**Comment by kichinsky** [comment]
Fixed "reset filters" issue in #274.

**Comment by kichinsky** [comment]
Hey, some updates here:
1. I'm going to add soon few basic filters (saturation, brightness, contrast).
2. As of now they will be exposed to the interface with prefifined params (like allow only "increase brightness by 25%"). So it is not customizable, but I hope it still adds some value.
3. FIlters are not combined into a pipeline. So adding a new filter, clears the previos one.
--
4. I think a more advanced UI for managing filters will be ready in parallel with a VoTT v.2 release.
